recipe/meta.yaml

```diff
@@ -136,6 +136,7 @@ outputs:
       script_env:
         - PY_INTERP_LINKAGE_NATURE={{ linkage_nature_env }}
         - PY_INTERP_DEBUG={{ py_interp_debug }}
+        - DEBUG_PY={{ py_interp_debug }}
```

Comment:
Why not just use `PY_INTERP_DEBUG`?

Reply:
Because `build_static.sh` needs `DEBUG_PY` and that's why the latest CI builds failed.

Reply:
Also because `build_base.sh` uses `DEBUG_PY` to add `--with-pydebug` to `./configure`.

Reply:
I could change those scripts to just rely on `PY_INTERP_DEBUG` instead.